js/router.js

```diff
@@ -38,6 +38,11 @@ define(['jquery', 'underscore', 'backbone', 'views/home/home', 'views/home/calen
 
   var initialize = function() {
       router = new AppRouter;
+
+      // Maybe this is the wrong place to call this, but 
+      // the hamburger menu applies everywhere.
+      hamburgerMenuView = new hamburgerMenuView();
```

Comment:
Please move this into views/home/home.js. Router shouldn't handle initializing views directly